internal/codegen/tpl_file.go

```diff
@@ -105,6 +110,24 @@ func (f *TplFile) Static() (out string, err error) {
 	return "", nil
 }
 
+// Once will only generate this file a single time, and store that fact
```

Comment:
Can you re-run the docs generation so this gets added to the docs? idr the command off the top of my head, but I believe I put it as a task in the mise task list

Reply:
talked offline.  works if i ghettorig it but needs fixing.